package.json

```diff
@@ -36,13 +36,13 @@
   "license": "W3C-20150513",
   "dependencies": {
     "@tpluscode/rdf-string": "^0.2.27",
-    "@types/rdf-js": "^4.0.2",
     "array-permutation": "^0.2.0",
     "n3": "^1.16.3",
-    "rdf-js": "^4.0.2",
     "yaml": "^2.2.1"
   },
   "devDependencies": {
+    "rdf-js": "^4.0.2",
+    "@rdfjs/types": "^1.1.0",
```

Comment:
`@rdfjs/types` should still be a dependency rather than a devDependency because you export interfaces/functions that use these types; and so you want `@rdfjs/types` to be available when downstream developers use rdfjs-c14n.